src/components/carmel-gearvr-controls.js

```diff
@@ -0,0 +1,200 @@
+var registerComponent = require('../core/component').registerComponent;
+var bind = require('../utils/bind');
+var getGamepadsByPrefix = require('../utils/tracked-controls').getGamepadsByPrefix;
+var THREE = require('../lib/three');
+
+var GAMEPAD_ID_PREFIX = 'Gear VR Touchpad';
+
+/**
+ * Vive Controls Component
+ * Interfaces with vive controllers and maps Gamepad events to
+ * common controller buttons: trackpad, trigger, grip, menu and system
+ * It loads a controller model and highlights the pressed buttons
+ */
+module.exports.Component = registerComponent('carmel-gearvr-controls', {
+  // since aframe-teleport-controls looks for tracked-controls, add one
+  dependencies: ['tracked-controls'],
```

Comment:
right, it is not a `tracked-controls` really, it only has a stub of that because others like `aframe-teleport-controls` are looking for entities with that label.  it makes `hand-controls` allow gaze-based usage on Carmel, so that things like `aframe-teleport-controls` can just reference `hand-controls` and not worry about specific implementation.  if you try https://chenzlabs.github.io/aframe-teleport-controls/mesh/ you can see what I mean (swipe and hold, then release).